clients/src/main/java/org/apache/kafka/clients/consumer/internals/ShareHeartbeatRequestManager.java

```diff
@@ -205,4 +208,25 @@ void reset() {
             }
         }
     }
+
+    @Override
```

Comment:
Please move this method earlier in the class, just after the constructors so the layout matches `ConsumerHeartbeatRequestManager`. This makes it easier for us to keep them in step over time.